src/envs/painting.py

```diff
@@ -500,7 +510,11 @@ def _get_tasks(self, num_tasks: int, num_objs_lst: List[int],
                 state.set(self._robot, "fingers", 0.0)
                 state.set(target_obj, "grasp", grasp)
                 state.set(target_obj, "held", 1.0)
-                assert self._OnTable_holds(state, [target_obj])
+                if CFG.env == "repeated_nextto_painting":
```

Comment:
why are we only doing this if repeated nextto painting? seems like a natural thing we'd want to do even in the normal painting?

Reply:
In the painting environment the held objects remained on the table. I had to add a change in z (to simulate lifting the object), because in the repeated_nextto_painting environment there are move options and if you move to another block the locations of the blocks would be the same.